### PR TITLE
E-core TMA update 3.51, Atom TMA update 1.2

### PR DESCRIPTION
**Pull request by rpmclaug**
E-core TMA update 3.51
Atom TMA update 1.2

### CODE REVIEW

**Comment by rpmclaug** [comment]
This should in theory resolve #96  's E-core/Atom TMA concern too.

File: Atom_TMA.csv [hunk 1]
```diff
@@ -47,8 +47,8 @@ Info.Core,UPI,,,,UOPS_RETIRED.ALL / INST_RETIRED.ANY,UOPS_RETIRED.ANY / INST_RET
...
 Info.L1_Bound,Address_Alias_Blocks,,,,100 * LD_BLOCKS.4K_ALIAS / MEM_UOPS_RETIRED.ALL_LOADS,100 * LD_BLOCKS.4K_ALIAS / MEM_UOPS_RETIRED.ALL_LOADS,,,Percentage of total non-speculative loads with a address aliasing block,,
 Info.L1_Bound,Load_Splits,,,,100 * MEM_UOPS_RETIRED.SPLIT_LOADS / MEM_UOPS_RETIRED.ALL_LOADS,100 * MEM_UOPS_RETIRED.SPLIT_LOADS / MEM_UOPS_RETIRED.ALL_LOADS,,,Percentage of total non-speculative loads that are splits,,
-Info.Inst_Mix,IpBranch,,,,INST_RETIRED.ANY / BR_INST_RETIRED.ALL_BRANCHES,INST_RETIRED.ANY / BR_INST_RETIRED.ALL_BRANCHES,,,Instructions per Branch (lower number means higher occurance rate), ,
-Info.Inst_Mix,IpCall,,,,INST_RETIRED.ANY / BR_INST_RETIRED.CALL,INST_RETIRED.ANY / BR_INST_RETIRED.CALL,,,Instruction per (near) call (lower number means higher occurance rate), ,
+Info.Inst_Mix,IpBranch,,,,INST_RETIRED.ANY / BR_INST_RETIRED.ALL_BRANCHES,INST_RETIRED.ANY / BR_INST_RETIRED.ALL_BRANCHES,,,Instructions per Branch (lower number means higher occurence rate), ,
```
Comment (edwarddavidbaker) [inline Atom_TMA.csv:50]: Sorry to request yet one more `occurance` change.

```
./E-core_TMA_Metrics.csv:64: occurence ==> occurrence
./E-core_TMA_Metrics.csv:65: occurence ==> occurrence
./Atom_TMA.csv:50: occurence ==> occurrence
./Atom_TMA.csv:51: occurence ==> occurrence
```

**Comment by edwarddavidbaker** [comment]
Thanks for submitting Atom TMA updates!
* ASCII checks look good.
* See notes for one minor spelling request.
* I diffd the output of Ian's perf converter script. The only changes are ADL description updates due to spelling fixes. This seems correct since MTL RWC would also depend on `TMA_Metrics-full.csv`.

**Comment by rpmclaug** [comment]
Good catch!

File: Atom_TMA.csv (hunk 1 above)
Comment (edwarddavidbaker) [reply Atom_TMA.csv]: Thanks Ryan.

**Review by edwarddavidbaker — APPROVED** [review]
LGTM.

**Pull request merged**